http/templates/bracket_detail.html.tera

```diff
@@ -40,12 +40,18 @@
                     </td>
                     <td>
                         {% if race.scheduled %}
-                            {{ race.scheduled }}
+                            Scheduled for {{ race.scheduled }}
+                        {% elif race.player_1.winner or race.player_2.winner %}
```

Comment:
i would probably make these stand out less. make the text smaller or lower contrast or something.

Reply:
Ok that makes 2 of us. I was messing around with text size here a looot and in the end I decided not to overthink it too much and push it as is. I'll post some variation screenshots on discord and we can pick one there.